src/OrchardCore.Modules/OrchardCore.Taxonomies/OrchardCore.Taxonomies.csproj

```diff
@@ -23,6 +23,7 @@
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Liquid.Abstractions\OrchardCore.Liquid.Abstractions.csproj" />
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.Navigation.Core\OrchardCore.Navigation.Core.csproj" />
     <ProjectReference Include="..\..\OrchardCore\OrchardCore.ResourceManagement\OrchardCore.ResourceManagement.csproj" />
+    <ProjectReference Include="..\OrchardCore.ContentLocalization\OrchardCore.ContentLocalization.csproj" />
```

Comment:
Why you didn't use `ContentLocalization.Abstraction`?

Reply:
I need ContentLocalization for _localizationEntries... No? Btw, what's the gain of Abstractions?

Reply:
We don't reference module implementation from another module, I think @deanmarcussen point to that below